recipe/meta.yaml

```
@@ -7,9 +7,11 @@ package:
 source:
   url: https://github.com/libsdl-org/SDL_ttf/archive/refs/tags/release-{{ version }}.tar.gz
   sha256: 2275d0ddfffa53f0efa628bc1621f662dacbd42467b5a44db99e38255fbb575a
+  patches:  # [osx]
+    - 349.patch  # [osx]
```

Please put the batch in a separate folder and document in the recipe where it comes from
```suggestion
  patches:
    # backport of https://github.com/libsdl-org/SDL_ttf/pull/349
    - patches/349.patch  # [osx]
```